### PR TITLE
Basic Player Sample App Update for version 2.0.0

### PR DESCRIPTION
**Pull request by cacheung**
Basic Player Sample App Update for version 2.0.0

- Update test app to work with ACPMedia version 2.0.0
- Add player state mute when volume is muted
- Updated Basic Player Sample app to fix the audit items.

## How Has This Been Tested?
Tested on iOS and Android.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by nporter-adbe — APPROVED** [review]
looks good, let's have Arjun take a look too.

File: sample/ReactNativeBasicPlayerSample/App.js [hunk 1]
```diff
@@ -254,6 +218,7 @@ export default class App extends Component {
     // Custom metadata keys
     mediaMetadata['isUserLoggedIn'] = 'false';
     mediaMetadata['tvStation'] = 'Sample TV station';
+    console.debug("just a text");
```
Comment (addb) [inline sample/ReactNativeBasicPlayerSample/App.js]: remove this log

File: sample/ReactNativeBasicPlayerSample/App.js [hunk 2]
```diff
@@ -293,9 +258,10 @@ export default class App extends Component {
     if (this.state.currentTracker === undefined) {
       console.log('tracker is null, cannnot track seek');
       return;
-    }
+    
     this.state.currentTracker.trackEvent(ACPMediaEvent.EventSeekStart);
   }
```
Comment (addb) [inline sample/ReactNativeBasicPlayerSample/App.js:262]: `this.state.currentTracker.trackEvent(ACPMediaEvent.EventSeekStart);`  should be after the `if` block.

**Review by addb — COMMENTED** [review]
Couple of fixes. Everything else looks good.

File: sample/ReactNativeBasicPlayerSample/App.js (hunk 1 above)
Comment (cacheung) [reply sample/ReactNativeBasicPlayerSample/App.js]: removed

File: sample/ReactNativeBasicPlayerSample/App.js (hunk 2 above)
Comment (cacheung) [reply sample/ReactNativeBasicPlayerSample/App.js:262]: updated

**Review by addb — APPROVED** [review]

**Pull request merged**